inst/private/python_copy_vars_to.m

```diff
@@ -28,13 +28,13 @@
     %% put inside try-except
     L = do_list(4, in, varargin);
     L = { 'try:' ...
-      sprintf('    %s = []', in) ...
-      L{:} ...
-      '    octoutput_drv("PYTHON: successful variable import")' ...
-      'except:' ...
-      '    octoutput_drv("PYTHON: Error in variable import")' ...
-      '    myerr(sys.exc_info())' ...
```

Comment:
The double output here is the bit that was causing #352 (and #118).  Basically I just remove one of them...